docs/platform-external/index.md

```diff
@@ -0,0 +1,21 @@
+# Overview
+
+<!-- > TODO/placeholder: provider overview of what Platform External is and goal of this section
+
+- What is Platform External?
+- When use Platform External?
+
+... -->
+
+The Platform `External` integration path allows third parties to self-serve and integrate with OpenShift without the need to modify any core payload components and without the need for direct involvement of OpenShift engineering.
```

Comment:
i don't think we need to put "External" in the block quotes. we can just say "Platform External", using capitalization for both words to indicate we are talking about this. we should reserve the block quotes for when we mean the actual value eg "set the platform type to `External`".